common/install.sh

```diff
@@ -119,7 +119,7 @@ cp_ch -n $INSTALLER/pix3lify.sh $UNITY$BINPATH/pix3lify
 log_handler "Using $BINPATH."
 sed -i -e "s|<MAGISK>|$MAGISK|" -e "s|<CACHELOC>|$CACHELOC|" -e "s|<BINPATH>|$BINPATH|" -e "s|<MODVERSION>|$(grep_prop versionCode $INSTALLER/module.prop)|" -e "s|<MODID>|$MODID|" $UNITY$BINPATH/icewizard
 if $MAGISK; then
-sed -i -e "s|<PROP>|$(echo $PROP)|" -e "s|<MODPATH>|/sbin/.magisk/img/Pix3lify|" -e "s|<MODPROP>|$(echo $MOD_VER)|" -e "s|<MOUNTPATH>|/sbin/.core/img|" $UNITY$BINPATH/pix3lify
+sed -i -e "s|<PROP>|$(echo $PROP)|" -e "s|<MODPATH>|/sbin/.magisk/img/Pix3lify|" -e "s|<MODPROP>|$(echo $MOD_VER)|" -e "s|<MOUNTPATH>|/sbin/.magisk/img|" $UNITY$BINPATH/pix3lify
```

Comment:
Won’t this drop support for older Magisk versions?

Reply:
Older than 18 yes. I'll see if i can change it to support older

Reply:
This module (https://github.com/null4n/vold-posix) has an example of how to detect different versions of Magisk, but you probably got an idea anyways :P.

Reply:
yep i already figured it out just havent pushed the changes yet. 

MAGISK_VERSIONCODE=$(echo $(get_file_value /data/adb/magisk/util_functions.sh "MAGISK_VERSIONCODE=") | sed 's|-.*||')

# Finding file values
get_file_value() {
	if [ -f "$1" ]; then
		cat $1 | grep $2 | sed "s|.*${2}||" | sed 's|\"||g'
	fi
}

if $MAGISK_VERSIONCODE -ge "18000"; then 
MODPATH=/sbin/.magisk/img
else
MODPATH=/sbin/.core/img
fi